js/compiler.ts

```diff
@@ -508,10 +563,13 @@ class Host implements ts.CompilerHost {
 // lazy instantiating the compiler web worker
 window.compilerMain = function compilerMain(): void {
   // workerMain should have already been called since a compiler is a worker.
-  window.onmessage = ({ data }: { data: CompilerReq }): void => {
+  window.onmessage = async ({ data }: { data: CompilerReq }): Promise<void> => {
     const { rootNames, configPath, config, bundle } = data;
-    const host = new Host(bundle);
+    util.log(">>> compile start", { rootNames, bundle });
+
+    await processImports(rootNames.map(rootName => [rootName, rootName]));
```

Comment:
Can you add a comment here why it's called? Took me a while to figure out that `processImports` is intentionally run before main compilation